pkg/cmd/roachtest/kv.go

```diff
@@ -576,7 +576,6 @@ func registerKVRangeLookups(r *testRegistry) {
 								kv.kv
 							SPLIT AT
 								VALUES (CAST(floor(random() * 9223372036854775808) AS INT))
-							WITH EXPIRATION '1s'
```

Comment:
Why did we remove this?

Reply:
Temporary splits weren't needed for the test and I feel like preventing automatic merges from taking place would probably be a good thing.